src/CachePoolTest.php

```diff
@@ -107,12 +105,38 @@ public function testBasicUsage()
         $this->assertFalse($this->cache->getItem('key2')->isHit());
     }
 
-    public function testItemModifiersReturnsStatic()
+    public function testBasicUsageWithLongKey()
     {
         if (isset($this->skippedTests[__FUNCTION__])) {
             $this->markTestSkipped($this->skippedTests[__FUNCTION__]);
+        }
+
+        $pool = $this->createCachePool();
+
+        $key = str_repeat('a', 300);
```

Comment:
This should never happen as per PSR-16, which has a maximum cap of 64 chars: https://www.php-fig.org/psr/psr-16/

Reply:
Actually, it is a minimum of `64 chars`.
```
Implementing libraries MAY support additional characters and encodings or longer lengths, but MUST support at least that minimum. 
```